README.md

```diff
@@ -7,6 +7,11 @@
 
 As of the v11.x.x release, babel-eslint now requires Babel as a peer dependency and expects a valid [Babel configuration file](https://babeljs.io/docs/en/configuration) to exist. This ensures that the same Babel configuration is used during both linting and compilation.
 
+Some node types has changes in accordance with changes in espree:
```

Comment:
I don’t think we actually need to put this in the readme since it’s only relevant to people upgrading to v11. (though I think it should be documented in a change log or release notes when v11 is finalized).

Reply:
Updated.